### ISSUE TITLE
Publish to Modrinth 

### ISSUE THREAD

**Issue by MimzyFang** [Alastors/Extended-Cogwheels-Unofficial#4]
Thanks again for reaching out to me and making me aware of this fork 

Though as I am planning to publish my modpack to Modrinth once I figured Branding out I would like to ask you to publish it there as then I would not need to add the file to my overwrites
Thanks in advance and have a nice day

**Comment by Karotte128** [comment]
This mod was on modrinth but got taken down for some reason

**Comment by MimzyFang** [comment]
Hmm I have a guess what happened

Modrinth staff might thought this is a hostile multiloader fork or something try appealing and talking to the Modrinth staff

**Comment by Alastors** [comment]
> Hmm I have a guess what happened
> 
> Modrinth staff might thought this is a hostile multiloader fork or something try appealing and talking to the Modrinth staff

yeah so basically I did this, but the original dev started trying to cease and desist me because his "assets were ARR," which they aren't, and we were able to prove this. But Modrinth started leaning on their "significant changes required to publish a fork unlike Curseforge, so I'm going to try republishing every version I make changes with.

**Comment by Karotte128** [comment]
why dont you make a pull request to the original repostitory?

**Comment by Alastors** [comment]
> why dont you make a pull request to the original repostitory?

Well man has finished PRs (from other people) on the mod that've been sitting since December which is why I made the unofficial fork in the first place, but frankly at this point the entire treatment of the situation just annoys me especially when he has multiple mods he's just straight abandoned like extended flywheels

**Comment by MimzyFang** [comment]
Upstream has actually been updated thanks for your work anyways @Alastors

**Comment by Alastors** [comment]
Terrific
